recipes/DABEST/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dabest" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8147e304ebc587923f0e2692ff68eb33e5abba81d0b93994da1d14cc4479dec4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
Are there any version restrictions on any of the `run` requirements? It looks like there are specific versions mentioned in the `setup.py`, but maybe they're not strict limits.

Reply:
it is for a few of the packages; should I add something?

Reply:
Sorry for the delay responding to this. If, for example, `seaborn` needed to strictly be at version `0.8`, you would change:

```
   - seaborn
```

to 

```
   - seaborn ==0.8
```

See examples in the following:

https://conda.io/docs/user-guide/tasks/build-packages/package-spec.html#examples

Reply:
OK, done!